src/env.h

```diff
@@ -701,6 +701,12 @@ class AsyncHooks : public MemoryRetainer {
   // The `js_execution_async_resources` array contains the value in that case.
   inline v8::Local<v8::Object> native_execution_async_resource(size_t index);
 
+  inline void SetJSPromiseHooks(
+    v8::Local<v8::Function> fn0,
+    v8::Local<v8::Function> fn1,
+    v8::Local<v8::Function> fn2,
+    v8::Local<v8::Function> fn3);
```

Comment:
I would update these to reflect the hook types also: init, before, after, resolve.

Reply:
Done.